client/main.js

```diff
@@ -34,10 +35,10 @@ window.addEventListener( 'beforeunload', ( event ) => {
 const rootEl = document.getElementById( 'wc-connect-admin-container' );
 
 let render = () => {
-	const WCCShippingSettings = require( 'views/shipping' );
+	const RootView = require( 'views/root/' + rootView );
```

Comment:
I don't think having a variable in your require is a good idea. It makes the code very unpredictable; I'm not even sure how WebPack works in this case...